profile.in

```diff
@@ -0,0 +1,13 @@
+# Default profile
+
+# replace built-in help command
+alias help="/usr/bin/help"
```

Comment:
need a way to call the built-in help too.

Reply:
Added `helpsh` for that.